buildtest/main.py

```diff
@@ -183,7 +182,11 @@ def main():
         if cmd.build_success():
             build_history_dir = cmd.get_build_history_dir()
 
-            shutil.move(fname, os.path.join(build_history_dir, "output.txt"))
+            shutil.copyfile(
```

Comment:
It might be a good idea to close the file before you copy it in case any write buffering or something else random happens.